content/docs/tasks/security/health-check/index.md

```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
+[SDS](/docs/tasks/security/auth-sds/) or [Mesh Expansion](/docs/setup/kubernetes/mesh-expansion/).
 
 ## Before you begin
 
 * Set up Istio by following the instructions in the
-  [quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
+  [Kubernetes quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
 
     {{< text bash >}}
     $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
     {{< /text >}}
 
     _**OR**_
 
-    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` to `true`.
+    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` set to `true`.
 
 {{< tip >}}
-Starting with Istio 0.7, you can use [authentication policy](/docs/concepts/security/#authentication-policies) to configure mutual TLS for all/selected services in a namespace (repeated for all namespaces to get global setting). See [authentication policy task](/docs/tasks/security/authn-policy/)
+You can use [authentication policy](/docs/concepts/security/#authentication-policies) to configure mutual TLS for
+all/selected services in a namespace (repeated for all namespaces to get global setting).
+See the [authentication policy task](/docs/tasks/security/authn-policy/) for details.
 {{< /tip >}}
 
 ## Deploying Citadel with health checking
 
-Deploy Citadel with health checking enabled.
+Redeploy Citadel with health checking enabled.
 
 {{< text bash >}}
 $ kubectl apply -f install/kubernetes/istio-citadel-with-health-check.yaml
 {{< /text >}}
 
-Deploy the `istio-citadel` service so that the CSR service can be found by the health checker.
-
-{{< text bash >}}
-$ kubectl create -f - <<EOF
-apiVersion: v1
-kind: Service
-metadata:
-  name: istio-citadel
-  namespace: istio-system
-  labels:
-    istio: citadel
-spec:
-  ports:
-    - port: 8060
-  selector:
-    istio: citadel
-EOF
-{{< /text >}}
-
 ## Verifying the health checker is working
 
 Citadel will log the health checking results. Run the following in command line:
 
 {{< text bash >}}
-$ kubectl logs `kubectl get po -n istio-system | grep istio-citadel | awk '{print $1}'` -n istio-system
+$ kubectl logs `kubectl get po -n istio-system | grep istio-citadel | awk '{print $1}'` -n istio-system | grep "CSR signing service"
 {{< /text >}}
 
 You will see the output similar to:
 
 {{< text plain >}}
-...
-2018-02-27T04:29:56.128081Z     info    CSR successfully signed.
-...
-2018-02-27T04:30:11.081791Z     info    CSR successfully signed.
-...
-2018-02-27T04:30:25.485315Z     info    CSR successfully signed.
-...
+... CSR signing service is healthy (logged every 100 times).
 {{< /text >}}
 
 The log above indicates the periodic health checking is working.
-Observe that the health checking interval is about 15 seconds, which is the default health checking interval.
+The health checking interval is (by default) 15 seconds. It is logged every 100 times.
```

Comment:
The phrase "is logged every 100 times" is ambiguous. Do you mean: 

1. The health check is logged once every 100 checks, or
1. The health check is logged only on for the 100th, 200th, etc... check?

Assuming 1:

```suggestion
The default health checking interval is 15 seconds and is logged once every 100 checks.
```